web/src/Msgs.elm

```diff
@@ -29,16 +31,18 @@ type Msg
     | SetVoteState VoteState ( VoteId, Vote )
     | SetBallotState BallotState ( BallotId, Ballot )
     | SetDelegateState DelegateState ( DemocracyId, Democracy )
-    | SetSelectField String (SelectWith SelectOptions Msg)
-    | Select String (SelectMsg SelectOptions)
+    | Debug String
     | MultiMsg (List Msg)
     | ChainMsgs (List Msg)
       --     Port Msgs
-    | Send SendMsg
-    | Receipt ( String, String )
-    | Confirm String
-    | Get String
-    | Receive String
+    | BlockchainSend SendMsg
+    | BlockchainReceipt ( String, String )
+    | BlockchainConfirm String
+    | BlockchainGet String
+    | BlockchainReceive String
+    | LocalStorageWrite { key : String, value : String }
+    | LocalStorageRead String
+    | LocalStorageReceive { key : String, value : String }
```

Comment:
This is good, only thing I'd be careful of is having too many msgs here.

For examples of what I think is better see:
https://github.com/swarmfund/sv-voting-mvp/blob/master/web/src/SecureVote/SPAs/SwarmMVP/Msg.elm#L44
https://github.com/swarmfund/sv-voting-mvp/blob/master/web/src/SecureVote/SPAs/SwarmMVP/Update.elm#L158

Basically break out msgs into subtypes (e.g. `ToBlockchain (ToBlockchainMsg)` and `FromBlockchain (FromBlockchainMsg)` - both of these are `Msg`s)